src/main/scala/org/apache/spark/sql/hive/llap/LlapExternalCatalog.scala

```diff
@@ -91,6 +92,32 @@ private[spark] class LlapExternalCatalog(
     }
   }
 
+  override def createTable(
+      tableDefinition: CatalogTable,
+      ignoreIfExists: Boolean): Unit = {
```

Comment:
The verification step is the same with `HiveExternalCatalog` [here](https://github.com/apache/spark/blob/master/sql/hive/src/main/scala/org/apache/spark/sql/hive/HiveExternalCatalog.scala#L189-L216). We need this to get only authorization exceptions in `stmt.executeUpdate(s"CREATE TABLE..`.